spec/livefeed/client_spec.rb

```diff
@@ -42,6 +42,11 @@ module Twingly::LiveFeed
           expect { subject }
             .to raise_error(AuthError, "No API key has been provided.")
```

Comment:
Same with this one. I think we can remove it now.

Reply:
71ce125